src/components/Patient/Utils.ts

```diff
@@ -17,3 +23,23 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+
+export const useAddPatientNote = (options: { patientId: string }) => {
+  const queryClient = useQueryClient();
+  const { patientId } = options;
+
+  return useMutation({
+    mutationFn: mutate(routes.addPatientNote, {
+      pathParams: { patientId },
+    }),
+    onSuccess: (data) => {
+      console.log("here");
+      queryClient.invalidateQueries({
+        queryKey: ["notes", patientId],
```

Comment:
do add the thread and consultation id too to the keys. why do we need to invalidate all threads when message is sent only to one thread?